src/providers/historical-telemetry-provider.js

```diff
@@ -40,48 +40,37 @@ export default class YamcsHistoricalTelemetryProvider {
         return this.getHistory(domainObject.identifier.key,
             options.start,
             options.end,
-            options.size);
+            options.size,
+            options.strategy);
     }
 
-    getHistory(id, start, end, size=300) {
+    getHistory(id, start, end, size=300, strategy) {
```

Comment:
We're getting a lot of parameters at this point along with some defaults, would an "options" paramter work here?